R-Script.R

```diff
@@ -0,0 +1,6 @@
+## function for adding 2 numbers
+
+add_2_numbers <- function(x, y){
```

Comment:
would call this function "add"

Reply:
i like this name, will merge. Thanks for the suggestion though :D